reference/Chronicler2.v0.yaml

```diff
@@ -138,102 +277,166 @@ components:
+                      nickname:
+                        type: string
+                      primaryColor:
+                        type: string
+                      secondaryColor:
+                        type: string
+                      shorthand:
+                        type: string
+                      slogan:
+                        type: string
+                  cancelled:
+                    type: boolean
+                  complete:
+                    type: boolean
+                  created:
+                    type: string
+                  day:
+                    type: integer
+                  deleted:
+                    type: boolean
+                  gameEventBatches:
                     type: array
                     items:
                       type: object
                       properties:
-                        name:
+                        batchData:
                           type: string
-                        value:
-                          type: number
-                  categoryRatings:
-                    type: array
-                    items:
-                      type: object
-                      properties:
-                        name:
+                        batchStep:
+                          type: integer
+                        gameId:
                           type: string
-                        stars:
-                          type:
-                            - integer
-                            - number
-                  id:
-                    type: string
-                  items:
-                    type: array
-                    items:
-                      type: object
-                      properties: {}
-                  modifications:
-                    type: array
-                    items:
-                      type: object
-                      properties: {}
-                  name:
+                  gameLoserId:
                     type: string
-                  overallRating:
-                    type: number
-                  playerHeatMaps:
-                    type: array
-                    items:
-                      type: object
-                      properties:
-                        currentValue:
-                          type: number
-                  positions:
+                  gameStates:
                     type: array
                     items:
                       type: object
                       properties:
-                        positionName:
+                        awayScore:
+                          type: integer
+                        balls:
+                          type: integer
+                        ballsNeeded:
+                          type: integer
+                        baserunners:
+                          type: array
+                          items:
+                            type: object
+                        batter:
                           type: 'null'
```

Comment:
(again, not super familiar with yaml, this might not be the way to do this/there might be a way of specifying more data)
````
type:
 - 'null'
 - object

Reply:
done